models/calendar.yml

```diff
@@ -130,6 +130,7 @@
         person:
           name: Alaina Kafkes
           twitter: alainakafkes
+          pronouns: # examples: she/her, they/them, he/him
```

Comment:
For discussion: Is this an adequate example placeholder? Is it sufficiently inclusive, and providing enough explanation?